dcist_launch_system/config_generation/experiment_overrides/relocalize_fused/roman_lc_node_overlay.yaml

```diff
@@ -0,0 +1,5 @@
+---
+/**:
+  ros__parameters:
+    prior_session_maps: !replace [$(env ADT4_PRIOR_MAP)/roman/]
```

Comment:
(minor) It might make sense to make the default relocalize config also do the loading from the directory (and then it would hopefully be transparent whether we're loading a fused or single-robot scene graph), but we can clean that up when we PR the fusion branch into main and get stuff working first